### ISSUE TITLE
[1.12.2]Recipes of Steel Structure Arm conflict with the Steel Scaffolding Stairs

### ISSUE THREAD

**Issue by lordfriend** [BluSunrize/ImmersiveEngineering#3261]
Immersive Engineering Version: 1.12.2-0.86
Minecraft Version: 1.12.2
Forge Version: 2768

Currently, a new but invisible block called Steel Structure Arm has the same recipes with Steel by default but has no use. I don't why this happen, and the tooltip shows it belongs to Immersive Engineering, So I think I should report at this repo.

For me, I just need to remove the recipes of Steel Structure Arm by using CraftTweak. But it's better to change the default recipes to avoid conflicts and allow users to use this mod with default settings.
![screenshot_786](https://user-images.githubusercontent.com/1235614/46710870-796d0900-cc7c-11e8-81a3-9817e088d899.png)

**Comment by lordfriend** [comment]
Yes, I have Optifine latest version, but not enable any shaderpack.

**Comment by malte0811** [comment]
I just tested with OF, they weren't invisible for me. Did you change any of the OF options from the default? What other mods do you use?

**Comment by lordfriend** [comment]
I don't think I change any settings in Optifine.
Here are my mods
```
[1.12]MoVillages-1.5.4.jar
BlackThorne+Civilian+Pack+[MTS]+1.10.2-11.0.0-1.1.3.jar
ChickenChunks-1.12.2-2.4.1.73-universal.jar
Chisel-MC1.12.2-0.2.1.34.jar
CodeChickenLib-1.12.2-3.2.2.353-universal.jar
CoFHCore-1.12.2-4.5.3.20-universal.jar
CraftTweaker2-1.12-4.1.10.jar
CreativeCore_v1.9.32_mc1.12.2.jar
CTM-MC1.12.2-0.3.2.18.jar
Decocraft-2.6.2_1.12.2.jar
Drones-0.3.0.jar
FTBLib-5.3.0.50.jar
FTBUtilities-5.3.0.49.jar
Hwyla-1.8.26-B41_1.12.2.jar
IGW-Mod-1.12.2-1.4.4-15-universal.jar
ImmersiveEngineering-0.12-85-core.jar
ImmersiveEngineering-0.12-86.jar
immersivepetroleum-1.12.2-1.1.9.jar
ImmersiveRailroading-1.3.3_1.12.jar
immersivetech-1.12-1.3.10.jar
IvToolkit-1.3.3-1.12.jar
jei_1.12.2-4.11.0.212.jar
journeymap-1.12.2-5.5.2.jar
leverbuttonlights-1.12.2-3.2.9.jar
LightingWand-1.0.4.jar
Minecraft+Transport+Simulator-1.12.2-11.2.0.jar
ModernLights-1.0.3_1.12.jar
MTS_Heavy_Industrial_by_AdamRK_1.8.0.jar
MTS_Official_Pack_V06.jar
OnlinePicFrame_v1.4.19_mc1.12.2.jar
OpenComputers-MC1.12.2-1.7.2.67.jar
OptiFine_1.12.2_HD_U_E2.jar
PTRLib-1.0.3.jar
RecurrentComplex-1.4.7.jar
RedstoneFlux-1.12-2.0.2.3-universal.jar
techguns-1.12.2-2.0.1.1 (1).jar
torohealth-1.12.2-11.jar
TrackAPI-1.1_1.12.jar
UNU+Parts+Pack+[MTS]+1.10.2-11.0.0-1.1.3.jar
Wawla-1.12.2-2.5.260.jar
worldedit-forge-mc1.12-6.1.8-dist.jar
```
All downloaded from curseforge

**Comment by FireheartIndustries** [comment]
I don't have optifine installed, but the block is invisible for me as well. I can post a list of installed mod to help find common mods to help track the problem down, but it will have to be later, since I'm at work right now.

That said, looking at the list above I know I have the following installed. 

Chisel
CodeChickenLib (and all requirements for NEI)
CTM
IE, IP, IT (Immersive Eng. Pet. Tech)
Journeymap
Recurrent Complex (recent install. Problem existed without it) 
Redstone Flux
WorldEdit

Will post full list later. Hope this helps

**Comment by malte0811** [comment]
Looks like it's CTM. I'm going to close this issue since the bug in the title has been fixed and #3266 is specifically about the rendering issue.
